undistort_test.py

```diff
@@ -0,0 +1,86 @@
+import open3d as o3d
+import argparse
+import numpy as np
+import mrob
+import math
+
+from pathlib import Path
+from scipy.spatial.transform import Rotation as R
+from collections import defaultdict
+
+def get_intermediate_transform(T1, T2, t):
```

Comment:
This is bad naming for variables, `T1` and `T2`. It's easy to create a typo using such naming